### PR TITLE
Indexer base codestyle

### PR DESCRIPTION
**Pull request by rybmat**
@sqreek

### CODE REVIEW

File: service/indexer-base/src/main/java/com/wikia/services/indexerbase/IndexerBaseApplication.java [hunk 1]
```diff
@@ -10,34 +10,34 @@
...
+  }
+
+  @Override
+  public void initialize(Bootstrap<IndexerConfiguration> bootstrap) {
+    name = System.getenv().get("APP_NAME");
+    super.initialize(bootstrap);
+  }
+
+  @Override
+  public void run(IndexerConfiguration configuration, Environment environment)
```
Comment (WikiaTech) [inline service/indexer-base/src/main/java/com/wikia/services/indexerbase/IndexerBaseApplication.java]: MAJOR Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)

File: service/indexer-base/src/main/java/com/wikia/services/indexerbase/guice/IndexerModule.java [hunk 2]
```diff
@@ -15,36 +15,36 @@
...
-                params.batchSize,
-                params.updatePeriod);
-    }
-
-    @Provides
-    DomainCache domainCache(IndexerConfiguration config) {
-        return config.getDomainCache();
-    }
+  @Override
+  protected void configure() {
```
Comment (WikiaTech) [inline service/indexer-base/src/main/java/com/wikia/services/indexerbase/guice/IndexerModule.java]: MAJOR Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 3 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Define and throw a dedicated exception instead of using a generic one.](https://github.com/Wikia/pandora/blob/5a56669a2626bf3c73af4babe1aa503c55bffac6/service/indexer-base/src/test/java/com/wikia/services/indexerbase/health/IndexerBaseHealthCheckTest.java#L19) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00112)

**Comment by sqreek** [comment]
LGTM

**Pull request merged**